src/Controller/Api/EntityController.php

```diff
@@ -197,14 +219,13 @@ public function handleDelete($entityClass, $id)
      * @param Request $request  The request object.
      * @param string  $method   The HTTP method.
      *
-     * @throws BadRequestHttpException When no valid parameters are passed.
-     * @throws BadRequestHttpException When invalid data is submitted.
+     * @throws BadRequestHttpException When there is an error.
```

Comment:
Should be only one @throws, or as many as different Exception cases can be thrown?

Reply:
I think we should have as many as Exceptions that are thrown

Reply:
It would be helpful in telling them apart